### DISCUSSION TITLE
[8.0 QA] Babel → SWC Upgrade

### DISCUSSION THREAD

**Discussion by vanessayuenn** [storybookjs/storybook#25683] [Announcement]
#### Feature owner: @valentinpalkovic 
Please comment in this discussion thread for any bug report and feedback for this feature as you test out Storybook 8. Kindly adhere to the following format:

> **Project**: Name of your project as entered in Notion
> **Version**: The version of storybook you tested with
> **Bug/feedback**: Details of the bugs/feedback you have

Feature owner should triage the comments here frequently and open issues as needed.

**Comment by shilman** [comment]
[The SWC docs](https://storybook.js.org/docs/8.0/api/main-config-swc) should emphasize up front that the setting is Webpack only. It mentions below that it only applies if `@storybook/addon-webpack5-compiler-swc` is enabled, but I think it should say this at the top, and make it more explicit.

**Reply by jonniebigodes** [reply]
@shilman appreciate the suggestion and I'll be more than glad to make the adjustments to make the documentation more explicit about this setting. I'll coordinate with Valentin on it.
